pkg/virt-api/webhooks/validating-webhook/admitters/pod-eviction-admitter_test.go

```diff
@@ -541,6 +486,26 @@ func newDeniedAdmissionResponse(message string) *admissionv1.AdmissionResponse {
 	}
 }
 
+func newExpectedPatch(vmi *virtv1.VirtualMachineInstance) testing.PatchActionImpl {
```

Comment:
the name newExpectedPatch sounds generic, but the patch changes something very specific. Please consider renaming the function to be more specific.

Reply:
Thank you for catching it.
I agree with you.
Done.